modules/prebidServerBidAdapter/ortbConverter.js

```diff
@@ -111,6 +113,19 @@ const PBS_CONVERTER = ortbConverter({
       bidResponse.adm = bidResponse.native.ortb;
     }
 
+    if (bidResponse.rendererurl) {
+      try {
+        bidResponse.renderer = Renderer.install({ url: bidResponse.rendererurl });
+        bidResponse.renderer.setRender((bid) => {
+          bidResponse.renderer.push(() => {
+            window.renderAd(bid);
```

Comment:
How do the renderers look like? are they used to this name, and how do they deal with clashes?

Reply:
I mentioned this in the original issue https://github.com/prebid/Prebid.js/issues/10434 - renderers will need to have a standard api. I don't think there is avoiding this. What do you mean by clashes?
```
It's important to note that supporting a dynamic renderer and incorporating bid renderer installation into the core of Prebid.js would necessitate a standardized API for each renderer. Currently, different adapters have varying methods for rendering an outstream video player.

An example of standardization for custom renderers for native ad units is in the linked documentation. The same API could be required in supporting this feature, that is required in the native ad unit case.

https://docs.prebid.org/prebid/native-implementation.html#43-implementing-the-custom-renderer-scenario
```